2-mandatory/4-space-colonies.js

```diff
@@ -15,7 +15,12 @@
   
 */
 
-function getSettlers() {}
+function getSettlers(array) {
+  let result = array.filter((num) => {
+    return num.includes("family") && num[0].startsWith("A");
+  });
+  return result;
+}
```

Comment:
This is good, using `.filter` here makes sense, only a small thing but I would just think about the variable names you're using sometimes, as it can make your code less readable. For example in the above solution you have used `num`, however each element in the array is actually a string, something like `settlerString` or even `family` might work better :)